hack/deploy-and-monitor-k8s-network-workload.sh

```diff
@@ -0,0 +1,237 @@
+  oc --ignore-not-found=true delete pod $POD_NAME
+  oc create -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: $POD_NAME
+  labels:
+    app: $POD_NAME
+spec:
+  containers:
+  - name: $POD_NAME
+$COMMON_POD_YAML
+    args:
+    - -c
+    - >
+$PRE_REQ
+      cd /tmp;
+      fallocate -l 20M /tmp/20MB.zip;
+      echo "=== starting network-workload ===";
+      while true; do
+        echo "Uploading (from Latvia, Riga)";
+        curl -T /tmp/20MB.zip http://bks4-speedtest-1.tele2.net/upload.php -O /dev/null || true;
+        echo "Done downloading.";
+        echo "Waiting $SLEEP seconds";
+        sleep $SLEEP;
+        echo "Done waiting.";
+      done ;
+      echo "=== Exiting ===";
+EOF
+  oc project default || true
+}
+
+deploy-pod-to-pod-workload() {
+  NAMESPACE=pod-to-pod-workload
+  POD_NAME=iperf-server
+  SLEEP=120
+  echo "====> Creating $NAMESPACE namespace"
+  oc create namespace $NAMESPACE || true
+  oc project $NAMESPACE || true
+  echo "====> Deploying $POD_NAME pod"
+  oc --ignore-not-found=true delete pod $POD_NAME
+  oc create -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: $POD_NAME
+  labels:
+    app: $POD_NAME
+spec:
+  containers:
+  - name: $POD_NAME
+$COMMON_POD_YAML
+    args:
+    - -c
+    - >
+$PRE_REQ
+      echo "=== starting iperf3 server ===";
+      iperf3 -s -p 3000 -f K
+      done ;
+      echo "=== Exiting ===";
+EOF
+  sleep 10
+  IPERF_SERVER_IP=$(oc get -n $NAMESPACE pod $POD_NAME -o jsonpath="{.status.podIP}")
+  echo "iperf server ip is $IPERF_SERVER_IP"
+
+  POD_NAME=iperf-client
+  echo "====> Deploying $POD_NAME pod"
+  oc --ignore-not-found=true delete pod $POD_NAME
+  oc create -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: $POD_NAME
+  labels:
+    app: $POD_NAME
+spec:
+  containers:
+  - name: $POD_NAME
+$COMMON_POD_YAML
+    args:
+    - -c
+    - >
+$PRE_REQ
+      cd /tmp;
+      echo "=== starting iperf client ===";
+      while true; do
+        echo "Stressing with iperf (20 seconds - 1 Mb/Sec)";
+        iperf3 -c $IPERF_SERVER_IP -p 3000 -f K -t 20 -b 1M;
+        echo "Done.";
+        echo "Waiting $SLEEP seconds";
+        sleep $SLEEP;
+        echo "Done waiting.";
+      done ;
+      echo "=== Exiting ===";
+EOF
+  oc project default || true
+}
+
+deploy-flowlogs2metrics() {
+  create-flowlogs2metrics-conf-file
+  echo "====> Deploying flowlogs2metrics project"
+  oc project default || true
+  oc project default
+  export FL2M_CONF_FILE=/tmp/flowlogs2metrics.conf.yaml
+  make ocp-deploy
+}
+
+main() {
+  echo ":::====> Start executing"
+  deploy-ingress-workload
+  deploy-egress-workload
+  deploy-pod-to-pod-workload
+  deploy-flowlogs2metrics
+  echo ":::====> Done executing"
+  echo "Use: kubectl logs -l app=flowlogs2metrics  | grep ingress-workload"
+  echo "Use: kubectl logs -l app=flowlogs2metrics  | grep egress-workload"
+  echo "Use: kubectl logs -l app=flowlogs2metrics  | grep pod-to-pod-workload"
+}
+
+main
```

Comment:
This is the main file of this PR ... it adds a test that deploys ingress, egress, and pod-to-pod in OCP and then deploys flowlogs2metrics to be able to see the flow-logs ... this is done without any aggregation. Just the pure raw flow logs (after we add k8s info to the logs)